pkgs/applications/networking/instant-messengers/zulip/default.nix

```diff
@@ -5,12 +5,12 @@
 
 let
   pname = "zulip";
-  version = "5.9.3";
+  version = "5.9.4";
   name = "${pname}-${version}";
```

Comment:
name is no longer required because appimageContents now takes pname + version.

Reply:
Opened #209557 for that followup.